kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java

```diff
@@ -48,6 +49,9 @@ private void start(String[] args) {
         try {
             // This is how you instantiate SPI while checking for JDK compatibility.
             spi = new Spi(posId, eftposAddress, spiSecrets); // It is ok to not have the secrets yet to start with.
+            spi.posVendorId = "assembly";
```

Comment:
This won't work with the latest code review changes in the library, you need to call `setPosInfo("assembly", "2.3.0");`

Reply:
Fixed